sql/core/src/main/scala/org/apache/spark/sql/DataStreamWriter.scala

```diff
@@ -84,6 +84,17 @@ final class DataStreamWriter private[sql](df: DataFrame) {
   }
 
   /**
+   * Specifies a name to the [[ContinuousQuery]] to be started. This name must be unique among
```

Comment:
It doesn't have to be exactly that text, but `Specifies a name to the ContinuousQuery to be started` doesn't sound right.  In other parts of the documentation we say "for the underlying datastream"

How about just "Specifies a name for this query."... and then stuff about auto assigning an uniqueness.